.circleci/config.yml

```diff
@@ -10,35 +10,62 @@ jobs:
     steps:
       - checkout
 
-      #TODO: add caching once demos share the same requirements file and can live in the same venv
-      #TODO: remove hard code in for-loops
+      # Restore cache
+      - run: &set-bash-env-template
+          name: Set bash environment
+          command: |
+            # Storing values shared between run calls
+            # Note: TEMP_FILE value is hardcoded in restore_cache and save_cache's checksum
+            echo 'export TEMP_FILE="temp_requirements_summary.txt"' >> $BASH_ENV
+            echo 'export IGNORED_DIRS="envs|tests|circuit-fault-diagnosis|factoring"' >> $BASH_ENV
+            . $BASH_ENV
```

Comment:
Why just not define these fixed/hardcoded variables in the CI script, using the [environment](https://circleci.com/docs/2.0/configuration-reference/#run) run key. See an [example](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-step).

Reply:
Okay, I will try that.

Reply:
It works beautifully as a job environment variable!